lib/assessment.rb

```diff
@@ -5,36 +5,52 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max {|a, b| a.length <=> b.length}
+    end
   end
 
+  puts longest
+
   # Question 2
   # Write ruby code to return the sum of entries in the
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce{|a, b| a + b}
```

Comment:
Correct (again ignoring the extra `end`)
Another way to write it would have been to use the `&` shortcut:
`enum.reduce(&:+)`
In fact, in Ruby 2.0 and above, you can even leave off the `&`:
`enum.reduce(:+)`